tests/common.go

```diff
@@ -4042,6 +4043,25 @@ func GetAzureCredsFromEnv() (tenantID, clientID, clientSecret, subscriptionID, a
 	return tenantID, clientID, clientSecret, subscriptionID, accountName, accountKey
 }
 
+// GetNfsCredsFromEnv get creds for nfs share.
+func GetNfsCredsFromEnv() (nfsServerAddress, nfsPath, nfsSubPath, nfsMountOptions string) {
+	nfsServerAddress = os.Getenv("NFS_SERVER_ADDR")
+	expect(nfsServerAddress).NotTo(equal(""),
```

Comment:
Don't use `expect`. I think we have purposely stopped using that.
You can use something like this
```
nfsServerAddress, isPresent := os.LookupEnv("NFS_SERVER_ADDR")
	if !isPresent {
                err := fmt.Errorf("Error message")
		log.FailOnError(err, "Some string")
	}
```

Reply:
Done